src/components/List/RowContent.js

```diff
@@ -170,8 +173,10 @@ const ListRowButton = StyledButton.withComponent("span").extend`
 
 const LinkRow = styled(Row)`
   padding-left: ${spacing.cozy};
+  padding-bottom: ${spacing.cozy};
   ${mediumAndUp`
-    padding-left: 12px;
+    padding-left: ${parseInt(CHEVRON_ICON_PADDING, 10) * 2 +
+      CHEVRON_ICON_SIZE}px;
```

Comment:
Can you please use a const with a type of string for this value? We should avoid converting strings to ints and performing arithmetic within these style blocks.

Reply:
parseInt() has been removed. Add calc() for defining padding-left based on ChevnokIcon size and padding. Is it ok? or should I add magic left-padding 47px?